doc/conf.py

```diff
@@ -302,3 +302,15 @@
     }
 
 numpydoc_class_members_toctree = False
+
+def touch_example_backreferences(app, what, name, obj, options, lines):
+    # generate empty examples files, so that we don't get
+    # inclusion errors if there are no examples for a class / module
+    examples_path = os.path.join(app.srcdir, "modules", "generated",
+                                 "%s.examples" % name)
```

Comment:
Should this be just ``os.path.join(app.srcdir, "generated",  "%s.examples" % name)``. It seems to fail on circle.